src/pages/home/report/ReportActionCompose.js

```diff
@@ -115,9 +115,26 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
+/**
+  * Return the max available index for arrow manager.
+ * @param {Number} numRows
+ * @param {Boolean} isEmojiPickerLarge
+ * @returns {Number}
+ */
+const getMaxArrowIndex = (numRows, isEmojiPickerLarge) => {
+    // EmojiRowCount is number of emoji suggestions. For small screen we can fit 3 items and for large we show up to 5 items
+    const emojiRowCount = isEmojiPickerLarge
+        ? Math.max(numRows, CONST.EMOJI_SUGGESTER.MAX_AMOUNT_OF_ITEMS)
+        : Math.max(numRows, CONST.EMOJI_SUGGESTER.MIN_AMOUNT_OF_ITEMS);
+
+    // -1 because we start at 0
+    return emojiRowCount - 1;
+};
+
 class ReportActionCompose extends React.Component {
     constructor(props) {
         super(props);
+        this.calculateEmojiSuggestionDebounce = _.debounce(this.calculateEmojiSuggestion, 10, false);
```

Comment:
NAB but typically we would just reuse the function name:

```suggestion
        this.calculateEmojiSuggestion = _.debounce(this.calculateEmojiSuggestion, 10, false);
```